client/my-sites/checkout/checkout/domain-details-form.jsx

```diff
@@ -178,6 +178,11 @@ export class DomainDetailsForm extends PureComponent {
 		this.setState( { currentStep: newStep } );
 	}
 
+	getDomainNames = () =>
```

Comment:
This seems pretty handy, we should probably add it to `client/lib/cart-values/cart-items.js`.